copulas/univariate/gaussian.py

```diff
@@ -14,54 +14,88 @@ class GaussianUnivariate(Univariate):
 
     def __init__(self):
         super().__init__()
-        self.column = None
+        self.name = None
         self.mean = 0
         self.std = 1
-        self.min = -np.inf
-        self.max = np.inf
 
     def __str__(self):
-        column_name = self.column.name if isinstance(self.column, pd.Series) else None
-        details = [column_name, self.mean, self.std, self.max, self.min]
+        details = [self.name, self.mean, self.std]
         return (
             'Distribution Type: Gaussian\n'
             'Variable name: {}\n'
             'Mean: {}\n'
-            'Standard deviation: {}\n'
-            'Max: {}\n'
-            'Min: {}'.format(*details)
+            'Standard deviation: {}'.format(*details)
         )
 
-    def fit(self, column):
-        if not len(column):
+    def fit(self, X):
+        """Fits the model.
+
+        Arguments:
+            X: `np.ndarray` of shape (n, 1).
+
+        Returns:
+            None
+        """
+
+        if not len(X):
             raise ValueError("Can't fit with an empty dataset.")
 
-        self.column = column
-        self.mean = np.mean(column)
-        std = np.std(column)
+        self.name = X.name if isinstance(X, (pd.Series, pd.DataFrame)) else None
```

Comment:
Write this as a multi-line if.
Use one liners only when checking simple values.